src/components/Homepage/HomepageHero.tsx

```diff
@@ -70,11 +81,11 @@ export const HomepageHero = () => {
         </div>
       </div>
       <m.div
-        animate={{ opacity: [0.5, 1, 1, 0] }}
-        transition={{ duration: 1.6, times: [0, 0.2, 0.9, 1], delay: 0.3 }}
-        className="su-cc su-opacity-50 su-absolute su-top-0 su-left-0 su-right-0 su-mb-0 su-pt-120 md:su-pt-216 2xl:su-pt-228 su-max-w-full"
+        animate={{ opacity: [1, 0] }}
+        transition={{ duration: 0.6, delay: 1.2 }}
+        className="su-cc su-absolute su-top-0 su-left-0 su-right-0 su-mb-0 su-pt-120 md:su-pt-216 2xl:su-pt-228 su-max-w-full"
       >
-        <Heading as="h1" size="f8" leading="none" font="druk" color="white">
+        <Heading as="h2" size="f8" leading="none" font="druk" color="white">
```

Comment:
I added a screen reader only H1 for the homepage, so this "How will we...." line becomes the h2